package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pinia-plugin-persistedstate",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
versions are managed automatically with the `release` script so no need to change it for now

Reply:
Ok, I undid the version change.